Std/CodeAction/Basic.lean

```diff
@@ -9,6 +9,31 @@ import Std.Lean.Name
 import Std.Lean.InfoTree
 import Std.CodeAction.Attr
 
+/-!
+# Helper constructors for WorkspaceEdit with document version
+
+In core lean, the version number is always 0.
+See Lean.Lsp.WorkspaceEdit.ofTextEdit in Lean.Data.Lsp.Basic.
+-/
+
+namespace Lean.Lsp.WorkspaceEdit
+
+/-- Construct a WorkspaceEdit from a VersionedTextDocumentIdentifier and a TextEdit --/
```

Comment:
This should be in `Std.Lean`, maybe `Std.Lean.Lsp`

Reply:
I think it may be better off in core as originally suggested by @bustercopley. Alternatively, if we keep it in std, we should delete `ofTextEdit` in core so that downstream users do not accidentally use the wrong function. I think I would personally prefer the former.

Reply:
SGTM

Reply:
Yes, especially since the changes to core amount to a simplificiation/streamlining, that is the natural place to do it. (I'm a bit intimidated by the process for core Lean PRs.)

Reply:
> Yes, especially since the changes to core amount to a simplificiation/streamlining, that is the natural place to do it. (I'm a bit intimidated by the process for core Lean PRs.)

It's fine to PR this without a corresponding RFC since it is very small, uncontroversial and you discussed it with the Lean community and the respective code owner :)

Reply:
@mhuisi :) Do you mind saying something along those lines in the Zulip topic?